client/src/components/useraccess/Register.jsx

```diff
@@ -0,0 +1,100 @@
+import React, {PropTypes} from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import Navbar from '../navbar/Navbar';
+import RegistrationForm from './Forms/RegistrationForm';
+import Background from '../Background/Background';
+import * as UserAcessActions from '../../Actions/userAccessAction';
+
+
+
+/**
+ * @class Register
+ * @classdesc returns the component for user signup
+ */
+class Register extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      userData: Object.assign({}, this.props.initialUserData),
```

Comment:
Uhmmnnn, I think this is an overkill, is `userData: this.props.initialUserData` not sufficient, after all, your first parameter is an empty object. And since you're not directly using the `=` you don't need to worry about both of them referencing the same location.